src/hotspot/share/runtime/mutex.hpp

```diff
@@ -98,6 +98,7 @@ class Mutex : public CHeapObj<mtSynchronizer> {
  protected:                              // Monitor-Mutex metadata
   PlatformMonitor _lock;                 // Native monitor implementation
   const char* _name;                     // Name of mutex/monitor
+  int _id;                               // ID for named mutexes
```

Comment:
It would be clearer if this were described in terms of how it is used - IIUC it is an index into the name array. `_id` is somewhat vague.

Reply:
Yes, it is an index into the `_name` array. I've added some comments.